psamm/datasource/sbml.py

```diff
@@ -713,20 +732,40 @@ def write_model(self, file, model, reactions, compounds):
 
         # Create list of reactions
         reactions = ET.SubElement(model_tag, self._sbml_tag('listOfReactions'))
-        for reaction in model.reactions:
+        # Create mapping for reactions containing flux limit definitions
+        for flux_limit in model.parse_limits():
+            eq_id = flux_limit[0]
+            # Deal with lower limit
+            if flux_limit[1] is None:
+                lower = (
+                    0
+                    if reaction_equations[eq_id].direction == Direction.Forward
+                    else -model.get_default_flux_limit())
```

Comment:
Using the inline `if` expression for this seems a bit odd because it is split over four lines. A normal `if` statement would probably be more clear.